enclave_manager/avalon_enclave_manager/avalon_enclave_helper.py

```diff
@@ -44,21 +44,15 @@ class EnclaveHelper(object):
 
     # -------------------------------------------------------
     @classmethod
-    def create_enclave_signup_data(cls, tcf_instance_keys=None):
-        """create_enclave_signup_data -- Create enclave signup data
-
-        :param tcf_instance_keys: Object of type TransactionKeys
+    def create_enclave_signup_data(cls, worker_mode):
+        """
+        create_enclave_signup_data -- Create enclave signup data
         """
-
-        if tcf_instance_keys is None:
-            tcf_instance_keys = keys.TransactionKeys()
```

Comment:
If TransactionKeys is not needed please remove this class

Reply:
Yes, will remove the class